.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature]"
```

Comment:
Instead of `[Feature]` I used `✨` as it’s visually more distinct:

```
[Feature] Lorem ipsum
[Bug] Lorem ipsum
```
Vs:
```
✨ Lorem ipsum 
🐞 Lorem ipsum
```

What do you think of this idea?

Reply:
I think that it is difficult to search but if we label them we can do in that way.

Reply:
Alright. I think that emojis are native to platform only on Mac so I agree